docs/internal-documentation/maintenance/ReleaseProcess-release.md

```diff
@@ -298,6 +298,9 @@ Create a new branch named `merge-develop-into-master` with both *master* and
 Create a pull request from `merge-develop-into-master` into `master` and have it
 merged through the normal process.
 
+Create a pull request from `master` on github back into `develop` and have it
+merged through the normal process.
+
```

Comment:
1. Is this new PR to be created after "`merge-develop-into-master` into `master`" has been merged, or can it be created before? That should be clarified.

2. This new text is at the end of a section with the header "Merge develop branch into master", and the section is really about that. I suggest that the new section is moved to a new section 17.